deps/libgit2.mk

```diff
@@ -80,7 +80,7 @@ endif
 
 clean-libgit2:
 	-$(MAKE) -C $(BUILDDIR)/$(LIBGIT2_SRC_DIR) clean
-	-rm -f $(LIBGIT2_OBJ_TARGET)
+	-rm -f $(build_shlibdir)/libgit2* $(build_shlibdir)/libssl* $(build_shlibdir)/libcrypt
```

Comment:
It tries to copy libssl and libcrypto and fails, because they are already there.